src/core/components/rerender/Rerender.test.tsx

```diff
@@ -0,0 +1,46 @@
+import '@testing-library/jest-dom';
+import { act, render, screen } from '@testing-library/react';
+import { Rerender } from './Rerender';
+
+jest.useFakeTimers();
```

Comment:
I would move this into a `beforeEach` function otherwise it might affect other tests